config/ext_private-nginx-ingress_publick8s.yaml

```diff
@@ -0,0 +1,6 @@
+controller:
+  replicaCount: 2
+  ingressClass: private-ingress
+  service:
+    annotations:
+      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: data-tier
```

Comment:
This answers my previous question about the creation of the corresponding subnet in azure-net

Reply:
Good catch. It meanse that this network must exist in order to have the internal LB created within.